app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  belongs_to :user, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
```

Comment:
- [ ] OPTIONAL
It is a nice idea to use one of the Rails Active Callbacks `after_save` to call the `update_posts_count method every time a new like has been saved to database, then make the `update_posts_count` method private, similar to:

```ruby
 # app/models/posts.rb
  after_save :update_post_count

  private

  def update_post_count
    author.increment!(:posts_count)
  end
```

Reply:
Hi @rloterh 

Thank you. I love the increment method above, it's quite preferable.